platooning_strategic/include/platoon_manager.hpp

```diff
@@ -100,46 +135,67 @@ namespace platoon_strategic
         int getNumberOfVehicleInFront();
         double getCurrentPlatoonLength();
         double getPlatoonRearDowntrackDistance();
-
-
+        
         double getDistanceFromRouteStart() const;
         double getDistanceToFrontVehicle();
         double getCurrentSpeed() const;
         double getCommandSpeed();
         double getCurrentDowntrackDistance() const;
 
 
-        int platoonSize;
-        std::string leaderID;
-        std::string currentPlatoonID;
-        bool isFollower;
+        int platoonSize = 2;
+        std::string leaderID = "";
+        std::string currentPlatoonID = "test_plan";
+        bool isFollower = false;
+
+        double current_speed_ = 0;
+        double command_speed_ = 0;
+
+        PlatoonState current_platoon_state = PlatoonState::STANDBY;
+
+        PlatoonPlan current_plan;
+
+        std::string targetLeaderId = "2";
```

Comment:
Can we make these default values more explicitly uninitalized? Some string like "INVALID" or "UNINTIALIZED" might make it more obvious that something has failed if these defaults ever make it to the display.